src/feature/dirauth/process_descs.c

```diff
@@ -108,8 +108,6 @@ add_rsa_fingerprint_to_dir(const char *fp, authdir_config_t *list,
   tor_strstrip(fingerprint, " ");
   if (base16_decode(d, DIGEST_LEN,
                     fingerprint, strlen(fingerprint)) != DIGEST_LEN) {
-    log_warn(LD_DIRSERV, "Couldn't decode fingerprint \"%s\"",
-             escaped(fp));
```

Comment:
Please don't delete this warning: you're hiding failures for every caller of this function.

Instead, do a minimal length check in dirserv_load_fingerprint_file().

Reply:
b6465c0aec08febe5d40dd343838095cb96faf1d